# Xu_2017

This repository includes code for Xu et al. 2017

#### Code:
**VCFtoTree can be accessed at https://github.com/duoduoo/VCFtoTree**   # A user friendly tool using 1000 genomes phase 3 data to build phylogeny.

**Ariadne_public.R**   # R code for plotting Figure 3A

**Figure3C.R**   # R code for plotting pairwise distance between haplogroups, Neanderthal, Denisovan and Chimpanzee.

**EX_0106.R**   # R code for plotting Figure 4C
**Simulation.R** # R code for plotting Figure 4C and Supplementary Figure 5

****




### The following items, please see:
https://drive.google.com/drive/folders/0B7vGA-5m4MLGdEVfZ0I2Z3AtQU0?usp=sharing

#### Alignments:
**muc7_aligned-repeat.fa**  # 1000 genomes phase 3 alignment for MUC7 region without repeats

**muc7_selfAligned_phase3_chimp_rhesus-repeat.FA**   # 1000 genomes phase 3 alignment for MUC7 region without repeats, with aligned Chimpanzee and Rhesus.

#### Supplementary tables:
**Supplementary Table 1. Human sample info.xlsx**  # Human sample population information for each individual, haplogroups and distribution across the world, and PCR copy number genotyping results etc.

**Supplementary Table 2. Simulation data.xlsx**  # The genetic variations observed in 10,000 simulated regions along the chromosome 4 and for MUC7.

**Supplementary Table 3. GWAS.xlsx**   # Accuracy for estimated haplogroup clustering and frequency of 5 copy allele in STAMPEED.

**Supplementary Table 4. microbiomeMUC7assoc.OTU.xlsx**   # Significant variants associated with microbiome

**Supplementary Table 5. Simulation.xlsx**   # Parameter values defined in simulations.
